### ISSUE TITLE
`version` is not being updated during publication

### ISSUE THREAD

**Issue by thewtex** [InsightSoftwareConsortium/itk-wasm#657]
The version update command:

https://github.com/InsightSoftwareConsortium/itk-wasm/blob/7ea56e32fc422af9b8d166c9ff3a988211de9a1a/package.json#L181

does not appear to be called during publication.

Checking via:

```
npm pack itk-wasm@1.0.0-b.32
tar xvzf itk-wasm-1.0.0-b.32.tgz
grep 'const version' ./package/dist/core/index.js
```

@floryst any ideas?

**Comment by floryst** [comment]
Either `core/index.js` loses it's version line, some bug in update-versions.cjs, or semantic-release/exec is not running the prepareCmd. I will try a local build to check the first two guesses.

**Comment by floryst** [comment]
So the update script is being run according to the logs, but I don't know if it's writing the correct result, or if it's getting overwritten somehow with the original version of the file. Running `update-versions.cjs` locally works.

**Comment by thewtex** [comment]
What do you think about changing the script execution to run as one of the Node publish hooks instead of semantic-release/exec?

**Comment by floryst** [comment]
That can work, so long as we can guarantee we can get the updated version (e.g. from package.json).

**Comment by thewtex** [comment]
#660 is an attempt to address this.

**Comment by thewtex** [comment]
Addressed via #668
